docs/man_pages/lib-management/plugin-create.md

```diff
@@ -20,9 +21,19 @@ The project is setup for easy commit in Github, which is why the command will as
 ### Options
 
 * `--path` - Specifies the directory where you want to create the project, if different from the current directory.
+* `--template` - Specifies the custom seed archive, which you want to use to create your plugin. If `--template` is not set, the NativeScript CLI creates the plugin from the default NativeScript Plugin Seed.
 * `--username` - Specifies the Github username, which will be used to build the URLs in the plugin's package.json file.
 * `--pluginName` - Used to set the default file and class names in the plugin source.
 
 ### Attributes
 
-* `<Plugin Repository Name>` is the name of repository where your plugin will reside. A directory with the same name will be created. For example: `nativescript-awesome-list`. If a directory with the name already exists and is not empty, the plugin create command will fail.
\ No newline at end of file
+* `<Plugin Repository Name>` is the name of repository where your plugin will reside. A directory with the same name will be created. For example: `nativescript-awesome-list`. If a directory with the name already exists and is not empty, the plugin create command will fail.<% if(isHtml) { %>
+* `<Template>` can be a URL or a local path to a `.tar.gz` file with the contents of a seed repository. This must be a clone of the [NativeScript Plugin Seed](https://github.com/NativeScript/nativescript-plugin-seed) and must contain a `src` directory with a package.json file and a script at `src/scripts/postclone.js`. After the archive is extracted, the postclone script will be executed with the username (`gitHubUsername`) and plugin name (`pluginName`) parameters given to the `tns plugin create` command prompts. For more information, visit the default plugin seed repository and [examine the source script](https://github.com/NativeScript/nativescript-plugin-seed/blob/master/src/scripts/postclone.js) there. Examples:
```

Comment:
`can be a URL or a local path to a .tar.gz` -> in fact it can be anything that can be npm installed, you can pass a local dir path with correct structure (i.e. local copy of the plugin seed for example) and it should work,

Reply:
I tried it and it only worked with .tar.gz archive. I suppose this is because the plugin seed does not have a package.json file in the root of the repository and it cannot be packed directly as an npm package.